core/src/test/java/dev/masonroot/nora/config/ConfigModel.java

```diff
@@ -0,0 +1,48 @@
+package dev.masonroot.nora.config;
+
+import java.util.Properties;
+import lombok.NonNull;
+
+public class ConfigModel implements Config {
```

Comment:
This class expresses what I discussed above. It highlights how there is a desire to have these precise configuration details yet they are all being obfuscated by a properties object. 

Set those properties free